tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -494,6 +496,9 @@ def _run(self):
                             )
                         vol_options.save()
 
+                # aggregate geo-replication status
+                georep_details.aggregate_session_status()
```

Comment:
Same as above,

if This is a cluster operation, it should be only run by the gluster-integration with tag "provisioner/$integration_id"